examples/axes_grid1/demo_axes_rgb.py

```diff
@@ -65,8 +62,6 @@ def demo_rgb():
 def demo_rgb2():
     fig = plt.figure(2)
```

Comment:
Can you remove the explicit figure number as well? It's better to use the auto-numbering unless there's good reason to work with explicit numbers.